### ISSUE TITLE
Todo List: Clarification on deleting Projects.

### ISSUE THREAD

**Issue by devnchill** [TheOdinProject/curriculum#29239]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

The Todo project doesn't explicitly mention handling project deletion. Should functionality to delete entire projects be included, or is it intentional that this feature isn't required? As I saw some solutions where people didn't add the option to delete projects .

### Path

Node / JS

### Lesson Url

https://www.theodinproject.com/lessons/node-path-javascript-todo-list

### (Optional) Discord Name

DevNChill

**Comment by rlmoser99** [comment]
It is intentional that deleting a project isn't required, so it would be expected to see projects that do not have this functionality. Just to make sure I am not missing something, is there anything that led you to think that this functionality should be part of the assignment?

**Comment by devnchill** [comment]
yes i will try to explain on what led me to think that it should be a part of it

- Let's say you create a project called blog , you list todos like add heading , work on conclusion ect ... . Now once user is done with it , he can mark it as complete and later delete it , but he won't be able to delete blog project . And this looks weird coz that project would be of no use but will still remain there as user doesn't have the delete option.

And yes of course, if the omission of a delete feature is intentional to keep the project scope focused, that's understandable. I just wanted to share my perspective on how it might impact usability. As so far I saw almost 10 different projects where they added deletion feature for project but I also saw 2,3 of them which didn't had this feature. 
I personally talked to them about this and that's when I noticed that this was indeed not specified as a requirement .

**Comment by devnchill** [comment]
Should I close this one ? Or does this need further discussion ?

**Comment by rlmoser99** [comment]
Thank you for taking the time to further explain your reasoning. I do not think we need to have additional discussion, as we intentionally do not include deleting a project as a requirement. However, I can see why people would choose to add this functionality, but I am hesitant to expand the scope of this project for everyone at this time.
